docs/multicast-guide.md

```diff
@@ -137,6 +137,13 @@ Support for Windows and IPv6 will be added in the future.
 Configuration option `multicastInterfaces` is not supported with encap mode.
 Multicast packets in encap mode are SNATed and forwarded to the transport interface only.
 
+### Large number of receiver groups for one node
+
+A Linux host limits the maximum number of multicast groups it can subscribe to
+by the number of [/proc/sys/net/ipv4/igmp_max_memberships](https://sysctl-explorer.net/net/ipv4/igmp_max_memberships/).
```

Comment:
Append "The default value is 20." after this sentence.

Reply:
updated